website/README.md

```diff
@@ -31,6 +32,44 @@ If you find a typo or you feel like you can improve the HTML, CSS, or JavaScript
 
 <!-- END: contributions -->
 
+## Backport labels
+
+The following backport labels exist to help facilitate backporting
+changes to `stable-website` and or `release-#.#.#` branches.
+
+### `backport/website`
+
+Merging a PR with this label will trigger two[^backport-strategy] workflow jobs:
+
+1. Attempt to backport changes into `stable-website`
+  - > **Note:** `stable-website` will represent the same version of the docs as `release-#.#.#`, not `main`.
+2. Attempt to backport changes into the "latest" release branch,
+   ex: `release-#.#.#`[^latest-release-branch]
+
+[^latest-release-branch]:
+    The _latest_ release branch is currently determined by the
+    greatest semver out of issue labels that match `backport/*`. See
+    [backport.yaml](../.github/workflows/backport.yaml) for implementation
+    details.
+
+[^backport-strategy]:
+    This is in support of versioned-documentation. When updating
+    docs content for `stable-website` it's assumed that the content changes
+    are intended for the current/latest version, and it makes sense to also
+    backport the same changes to the associated release branch. This ensures
+    that content doesn't get lost in the event that `stable-website` is
+    forced-pushed over.
+
+### `backport/#.#.#`
+
+Merging a PR with this will trigger a workflow job that
+attempts to backport changes to `release-#.#.#`.
```

Comment:
The labels we currently have are for things like `backport/1.1.x`. Does this backport to all the 1.1.x docs? Do we need a label for each specific version we want to backport to?

Reply:
@tgross sorry for totally losing track of this.

> Does this backport to all the 1.1.x docs?

No

The [current label implementation](https://github.com/hashicorp/nomad/blob/main/.github/workflows/backport.yml#L41-L47) is a 1-to-1, label to target branch mapping, with the exception of `backport/website` which targets  1.) `stable-website` and 2.) the _latest_ `release-...` branch.

> Do we need a label for each specific version we want to backport to?

Yes

We can look into a [matrix](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) behavior ( 1 label to many targets ) if you'd like? / If your team would benefit from this?

Reply:
> We can look into a matrix behavior ( 1 label to many targets ) if you'd like?

I think the tricky bit is that we want to use this for code backporting as well, so if we did something like that it'd have to be configurable differently for docs vs non-docs PRs. Probably not worth the confusion to do so.

Reply:
(Just so I fully understand...)

> I think the tricky bit is that we want to use this for code backporting as well

`backport-assistant` technically doesn't discriminate between if a PR is "docs" vs "code" vs "other", so it _would work_ right now.

(...With this approach: The matrix workflow would run multiple `backport-assistant`s — 1 for each release branch that falls under a specified minor version:

| Label | Targets |
| :-- | :-- |
| `backport/1.2.3` | `release-1.2.3` |
| `backport/1.2` | `release-1.2.0`, `release-1.2.1`, `release-1.2.2`, `release-1.2.3` |


> so if we did something like that it'd have to be configurable differently for docs vs non-docs PRs.

Are you saying that the backporting logic **would** need to know the difference between a "docs" or "non-docs" PR?

Reply:
Sorry for being unclear. We currently use `backport/1.2` to mark code PR backports for "whatever version we'll be releasing in the 1.2.x series next". Configuring the backport assistant to do a matrix backport for docs would be very confusing, because `backport/1.2` would mean two different things, depending on whether it was something that the backport assistant was handling or not.

Presumably once we have backport assistant configured for non-docs PRs, we'll want to abandon the `backport/1.2` labels in lieu of specific versions where we're marking backports to unrelease branches or something. But for now, there's nothing to do until we're using backport assistant across the whole codebase.